bootstrapper/cmd/bootstrapper/main.go

```diff
@@ -157,16 +160,28 @@ func main() {
 			metadata, measurements, helmClient, &kubewaiter.CloudKubeAPIWaiter{},
 		)
 
-		openTPM = vtpm.OpenVTPM
+		openDevice = vtpm.OpenVTPM
 		fs = afero.NewOsFs()
 
 	case cloudprovider.QEMU:
-		measurements, err := vtpm.GetSelectedMeasurements(vtpm.OpenVTPM, vtpm.QEMUPCRSelection)
-		if err != nil {
-			log.With(zap.Error(err)).Fatalf("Failed to get selected PCRs")
-		}
+		var measurements measurements.M
+		if tdx.Available() {
+			measurements, err = tdx.GetSelectedMeasurements(tdxapi.Open, []int{0, 1, 2, 3, 4})
+			if err != nil {
+				log.With(zap.Error(err)).Fatalf("Failed to get selected RTMRs")
+			}
+			issuer = tdx.NewIssuer(log)
```

Comment:
Shouldn't other issuers also have logs? 
If this is solely for debugging and to be removed later, maybe marking a `TODO` might be useful to spot this in later reviews?

Reply:
:wink: 
https://github.com/edgelesssys/constellation/pull/1264